bin/maintenance/update_header.py

```diff
@@ -173,36 +161,46 @@ def blacklisted(root, path, _cache={}):
     return _cache[orig_path]
 
 
-def main():
-    if '-h' in sys.argv[1:] or '--help' in sys.argv[1:] or not sys.argv[1:]:
-        print(USAGE)
+@click.command(help=USAGE)
+@click.option('--ci', is_flag=True, help='Indicate that the script is running during CI and should use a non-zero '
+                                         'exit code unless all headers were already up to date.')
+@click.argument('project', nargs=1)
+@click.argument('year', nargs=1, required=False, callback=_process_year)
+@click.argument('path', nargs=1, required=False, type=click.Path(exists=True))
```

Comment:
What about making these two args options instead of arguments? Having them as positional arguments isn't particularly intuitive anyway.

Reply:
I was wondering the same previously.